[2021-04-25 14:17:16,455] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: templated_bash_operator_dag.read_json_file_task 2021-04-25T14:17:13.857032+00:00 [queued]>
[2021-04-25 14:17:16,501] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: templated_bash_operator_dag.read_json_file_task 2021-04-25T14:17:13.857032+00:00 [queued]>
[2021-04-25 14:17:16,501] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 14:17:16,502] {taskinstance.py:1069} INFO - Starting attempt 1 of 3
[2021-04-25 14:17:16,502] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 14:17:16,544] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): read_json_file_task> on 2021-04-25T14:17:13.857032+00:00
[2021-04-25 14:17:16,554] {standard_task_runner.py:52} INFO - Started process 133 to run task
[2021-04-25 14:17:16,569] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'templated_bash_operator_dag', 'read_json_file_task', '2021-04-25T14:17:13.857032+00:00', '--job-id', '934', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bash_operator/templated_bash_operator_dag.py', '--cfg-path', '/tmp/tmpute5rk3n', '--error-file', '/tmp/tmpycgxwwbn']
[2021-04-25 14:17:16,570] {standard_task_runner.py:77} INFO - Job 934: Subtask read_json_file_task
[2021-04-25 14:17:16,698] {logging_mixin.py:104} INFO - Running <TaskInstance: templated_bash_operator_dag.read_json_file_task 2021-04-25T14:17:13.857032+00:00 [running]> on host d6e2455ef0ec
[2021-04-25 14:17:16,859] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=templated_bash_operator_dag
AIRFLOW_CTX_TASK_ID=read_json_file_task
AIRFLOW_CTX_EXECUTION_DATE=2021-04-25T14:17:13.857032+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-04-25T14:17:13.857032+00:00
[2021-04-25 14:17:16,867] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-04-25 14:17:16,873] {bash.py:158} INFO - Running command: 
    "execution date: 2021-04-25"
    echo todo.json
    
[2021-04-25 14:17:16,933] {bash.py:169} INFO - Output:
[2021-04-25 14:17:17,009] {bash.py:173} INFO - bash: line 1: execution date: 2021-04-25: command not found
[2021-04-25 14:17:17,010] {bash.py:173} INFO - todo.json
[2021-04-25 14:17:17,015] {bash.py:177} INFO - Command exited with return code 0
[2021-04-25 14:17:17,106] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=templated_bash_operator_dag, task_id=read_json_file_task, execution_date=20210425T141713, start_date=20210425T141716, end_date=20210425T141717
[2021-04-25 14:17:17,194] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-04-25 14:17:17,229] {local_task_job.py:146} INFO - Task exited with return code 0
